ball_pit/ball_pit.ino

```diff
@@ -372,6 +372,34 @@ void reset_timer() {
   target_failed = false;
 }
 
+/////////////////
+// COMPARISONS //
+/////////////////
+
+bool is_power_of_two(unsigned long number){
+  while (((number & 1) == 0) && number > 1) {
+    number >>= 1;
+  }
+  return (number == 1);
+}
+
+bool is_monodigit(unsigned long number){
+  char string[10]; // Since the max of an unsigned long is 4,294,967,295 this buffer is large enough for any number.
```

Comment:
Also, avoid using `string` as a name as it could get kind of confusing (even though there isn't a string type). Maybe `number_string`?